packages/create-contentful-app/lib/create-app-definition.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-console, no-process-exit */
+
+const inquirer = require('inquirer');
+const path = require('path');
+const { createClient } = require('contentful-management');
+const login = require('./login');
+
+async function fetchOrganizations(client) {
+  try {
+    const orgs = await client.getOrganizations();
+
+    return orgs.items.map(org => ({
+      name: `${org.name} (${org.sys.id})`,
+      value: org.sys.id
+    }));
+  } catch (err) {
+    console.log();
+    console.log(
+      'We couldnt find your organizations. Please make sure you provided an access token'
+    );
+    console.log();
+    console.log(err.message);
+    process.exit(1);
+  }
+}
+
+module.exports = async function createAppDefition(appDefinitionSettings = {}) {
+  let selectedOrg;
+
+  // login and create credentials file
```

Comment:
do we create the file? IMO if we can skip it we should

Reply:
Not anymore - leftovers from previous design